platform/kobo/koreader.sh

```diff
@@ -217,6 +217,17 @@ if [ -n "${ORIG_FB_BPP}" ]; then
     ./fbdepth -d "${ORIG_FB_BPP}" >>crash.log 2>&1
 fi
 
+# Restore original fb rotation on the Forma, for KSM
+if [ "${PRODUCT}" = "frost" ]; then
+    # Only needed for KSM, pickel -> Nickel will restore its own rota properly
+    if [ "${FROM_NICKEL}" != "true" ]; then
+        # The Forma kernel inverts odd rotation constants, counteract that, àla Plato because one-liner ;p.
```

Comment:
I'm oddly bothered by that à la without a space. :-P

Reply:
That's my own bad habit, AFAICT ;).